criu/cr-restore.c

```diff
@@ -3133,6 +3133,9 @@ rst_prep_creds_args(CredsEntry *ce, unsigned long *prev_pos)
 {
 	unsigned long this_pos;
 	struct thread_creds_args *args;
+	int n, len;
```

Comment:
`n` and `len` are much too far away from point of use to be named so simply.

Reply:
Why are they declared so far away though? Is it because one of the previous PR comments was to declare at the top of the function? This very function has a number of variables declared in a number of inner scopes.

Reply:
Yeah, but you need a scope to declare it in. The variables has to be at the top of the scope and that is the top of the scope in which they are used.

Reply:
I can rename them to accommodate for the distance.

Reply:
No need if you move the code to a separate function